File: testdata/csharp/Simple.cs
using System;
using System.Collections.Generic;

namespace MyApp.Models
{
    public interface IGreeter
    {
        string Greet(string name);
    }

    public enum Priority
    {
        Low,
        Medium,
        High
    }

    public class SimpleGreeter : IGreeter
    {
        public string Greet(string name)
        {
            return $"Hello, {name}!";
        }

        private void LogGreeting(string message)
        {
            Console.WriteLine(message);
        }
    }

    public static class Program
    {
        public static void Main(string[] args)
        {
            var greeter = new SimpleGreeter();
            var result = greeter.Greet("World");
            Console.WriteLine(result);
        }
    }
}

File: testdata/kotlin/Simple.kt
package com.example

import com.example.utils.Helper

interface Greeter {
    fun greet(name: String): String
}

class SimpleGreeter : Greeter {
    override fun greet(name: String): String {
        return "Hello, $name!"
    }

    private fun logGreeting(message: String) {
        println(message)
    }
}

object AppConfig {
    val defaultName = "World"
}

fun main() {
    val greeter = SimpleGreeter()
    val result = greeter.greet(AppConfig.defaultName)
    println(result)
    Helper.assist()
}
